models/event.js

```diff
@@ -0,0 +1,12 @@
+var mongoose = require('mongoose');
+
+var Event = new mongoose.Schema({
+	id: String,
```

Comment:
Minor - I think MongoDB uses a auto generated `_id` field for documents to ensure uniqueness. Since we are already providing our own unique id, `_id` field seems unnecessary. 

Is there a way to remove the default `_id` field? Don't bother if it is too much hassle.

Reply:
I think it can't be removed ([MongoDB docs](https://www.mongodb.com/docs/manual/core/document/#the-_id-field)). I could set `_id` to be the same value as our own `id`, but I don't see that it makes a difference

Reply:
I would agree with Pallavi. If we could avoid creating another ID, it might be beneficial. Could we use our generated `id` to be used as `_id` so we don't have a duplicated field serving the same purpose?

Reply:
MongoDB docs (linked above) says that `_id` is always present. I can't find any hint that there are any exceptions or ways to change this

I also don't see what the benefit is of removing the `_id`

Reply:
Just checked, if we provide our own `_id` field MongoDB does not creates one. So we just need to change our field name to `_id`.
https://www.mongodb.com/docs/manual/core/document/#the-_id-field

Reply:
OK, do you want both `_id` and `id` or to replace `id` with `_id`? I still don't quite understand what the problem is

Reply:
`id` and `_id` have duplicated purpose. Use `_id` only by injecting `UUID inside so MongoDB does not auto-generate it.

Reply:
The purpose of us providing our own `id` field was to be able to uniquely identify events. Since MongoDB already supports `_id` field which does the same thing as our `id` field, it makes sense to just have one field.

Since it is not possible to get rid of `_id` field, lets just use it. However, we want to be able to provide our own value instead of MongoDB auto-generating it. MongoDB will not generate `_id` field if we provide one.

So, in short we replace `id` with `_id` :)

Reply:
Done